### ISSUE TITLE
numpy no-member errors in test suite

### ISSUE THREAD

**Issue by bnavigator** [PyCQA/pylint#4877]
### Bug description

Running the test suite for 2.9.6 or the main branch results in test failures for Python 3.8 and Python 3.9 (but not for Python 3.6)

### Command used

```shell
pytest --benchmark-disable
```

### Pylint output

```
[  162s] =================================== FAILURES ===================================
[  162s] _________________________ test_functional[len_checks] __________________________
[  162s] 
[  162s] self = <pylint.testutils.lint_module_test.LintModuleTest object at 0x7fb71e69bc70>
[  162s] 
[  162s]     def runTest(self):
[  162s] >       self._runTest()
[  162s] E       AssertionError: Wrong results for file "len_checks":
[  162s] E       
[  162s] E       Unexpected in testdata:
[  162s] E        134: no-member
[  162s] E       
[  162s] E       Actual pylint output for this file:
[  162s] E       OutputLine(symbol='len-as-condition', lineno=4, column='3', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=7, column='3', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=11, column='9', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=14, column='11', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=55, column='5', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=60, column='5', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=63, column='6', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=66, column='6', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=69, column='12', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=72, column='6', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=95, column='11', object='github_issue_1331_v2', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=98, column='11', object='github_issue_1331_v3', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=101, column='17', object='github_issue_1331_v4', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=103, column='9', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=104, column='9', object='', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=124, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=125, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=126, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=127, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=128, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=129, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=130, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=134, column='18', object='github_issue_1879', msg="Module 'numpy' has no 'array' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='len-as-condition', lineno=171, column='11', object='github_issue_1879', msg='Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty', confidence='HIGH')
[  162s] E       OutputLine(symbol='undefined-variable', lineno=183, column='11', object='github_issue_4215', msg="Undefined variable 'undefined_var'", confidence='HIGH')
[  162s] E       OutputLine(symbol='undefined-variable', lineno=185, column='11', object='github_issue_4215', msg="Undefined variable 'undefined_var2'", confidence='HIGH')
[  162s] 
[  162s] ../../BUILDROOT/python-pylint-3.0.0~dev0-0.x86_64/usr/lib/python3.9/site-packages/pylint/testutils/lint_module_test.py:80: AssertionError
[  162s] ____________________ test_functional[nan_comparison_check] _____________________
[  162s] 
[  162s] self = <pylint.testutils.lint_module_test.LintModuleTest object at 0x7fb71d741220>
[  162s] 
[  162s]     def runTest(self):
[  162s] >       self._runTest()
[  162s] E       AssertionError: Wrong results for file "nan_comparison_check":
[  162s] E       
[  162s] E       Unexpected in testdata:
[  162s] E          6: no-member
[  162s] E          7: no-member
[  162s] E          9: no-member
[  162s] E         12: no-member
[  162s] E         13: no-member
[  162s] E         15: no-member
[  162s] E         17: no-member
[  162s] E         19: no-member
[  162s] E         21: no-member
[  162s] E       
[  162s] E       Actual pylint output for this file:
[  162s] E       OutputLine(symbol='nan-comparison', lineno=6, column='4', object='', msg="Comparison 'x is numpy.NaN' should be 'math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=6, column='9', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=7, column='4', object='', msg="Comparison 'x == numpy.NaN' should be 'math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=7, column='9', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=8, column='4', object='', msg="Comparison 'x == float('nan')' should be 'math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=9, column='4', object='', msg="Comparison 'numpy.NaN == numpy.NaN' should be 'math.isnan(numpy.NaN)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=9, column='4', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='no-member', lineno=9, column='17', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=12, column='4', object='', msg="Comparison 'numpy.NaN is not x' should be 'not math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=12, column='4', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=13, column='4', object='', msg="Comparison 'numpy.NaN != x' should be 'not math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=13, column='4', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=15, column='4', object='', msg="Comparison 'x != numpy.NaN' should be 'not math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=15, column='9', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=16, column='5', object='', msg="Comparison 'x != float('nan')' should be 'not math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=17, column='7', object='', msg="Comparison 'x == numpy.NaN' should be 'math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=17, column='12', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=18, column='7', object='', msg="Comparison 'x is not float('nan')' should be 'not math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=19, column='3', object='', msg="Comparison 'x == numpy.NaN' should be 'math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=19, column='8', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] E       OutputLine(symbol='nan-comparison', lineno=21, column='9', object='', msg="Comparison 'x is numpy.NaN' should be 'math.isnan(x)'", confidence='HIGH')
[  162s] E       OutputLine(symbol='no-member', lineno=21, column='14', object='', msg="Module 'numpy' has no 'NaN' member", confidence='INFERENCE')
[  162s] 
[  162s] ../../BUILDROOT/python-pylint-3.0.0~dev0-0.x86_64/usr/lib/python3.9/site-packages/pylint/testutils/lint_module_test.py:80: AssertionError
```

### Expected behavior

successful test suite

### Pylint version

pylint 2.9.6 with patch #4816 OR pylint main branch
astroid 2.7.1

### OS / Environment

openSUSE Tumbleweed build environment

### Additional dependencies

```
[    1s] [1/250] keeping compat-usrmerge-tools-84.87-3.1
[    1s] [2/250] keeping file-magic-5.40-1.13
[    1s] [3/250] keeping kernel-obs-build-5.13.8-1.1
[    1s] [4/250] keeping libsemanage-conf-3.2-1.14
[    1s] [5/250] keeping libssh-config-0.9.5-1.4
[    1s] [6/250] keeping pkgconf-m4-1.7.4-1.1
[    1s] [7/250] keeping python-rpm-macros-20210628.eccf3f2-1.1
[    1s] [8/250] keeping system-user-root-20190513-1.35
[    1s] [9/250] keeping vim-data-common-8.2.3318-1.1
[    1s] [10/250] keeping filesystem-84.87-2.1
[    1s] [11/250] keeping glibc-2.33-9.1
[    1s] [12/250] keeping fdupes-2.1.2-1.15
[    1s] [13/250] keeping fillup-1.42-276.6
[    1s] [14/250] keeping libacl1-2.3.1-1.13
[    1s] [15/250] keeping libatomic1-11.1.1+git536-1.1
[    1s] [16/250] keeping libattr1-2.5.1-1.12
[    1s] [17/250] keeping libaudit1-3.0.3-1.1
[    1s] [18/250] keeping libblkid1-2.36.2-2.1
[    1s] [19/250] keeping libbrotlicommon1-1.0.9-1.4
[    1s] [20/250] keeping libbz2-1-1.0.8-3.1
[    1s] [21/250] keeping libcap-ng0-0.7.10-1.29
[    1s] [22/250] keeping libcap2-2.51-1.1
[    1s] [23/250] keeping libcom_err2-1.46.3-1.1
[    1s] [24/250] keeping libcrypt1-4.4.23-1.1
[    1s] [25/250] keeping libeconf0-0.4.1+git20210709.cf671f2-1.1
[    1s] [26/250] keeping libexpat1-2.4.1-1.1
[    1s] [27/250] keeping libffi8-3.3.git30-1.27
[    1s] [28/250] keeping libgcc_s1-11.1.1+git536-1.1
[    1s] [29/250] keeping libgdbm6-1.19-1.15
[    1s] [30/250] keeping libgmp10-6.2.1-3.13
[    1s] [31/250] keeping libgomp1-11.1.1+git536-1.1
[    1s] [32/250] keeping libgpg-error0-1.42-1.12
[    1s] [33/250] keeping libitm1-11.1.1+git536-1.1
[    1s] [34/250] keeping libkeyutils1-1.6.3-2.1
[    1s] [35/250] keeping liblua5_4-5-5.4.3-4.1
[    1s] [36/250] keeping liblzma5-5.2.5-2.1
[    1s] [37/250] keeping libmnl0-1.0.4-2.6
[    1s] [38/250] keeping libnghttp2-14-1.43.0-1.4
[    1s] [39/250] keeping libopenssl1_1-1.1.1k-3.1
[    1s] [40/250] keeping libpcre1-8.45-1.1
[    1s] [41/250] keeping libpcre2-8-0-10.37-1.1
[    1s] [42/250] keeping libpkgconf3-1.7.4-1.1
[    1s] [43/250] keeping libpopt0-1.18-2.16
[    1s] [44/250] keeping libsasl2-3-2.1.27-5.4
[    1s] [45/250] keeping libsepol2-3.2-3.1
[    1s] [46/250] keeping libsmartcols1-2.36.2-2.1
[    1s] [47/250] keeping libtextstyle0-0.21-2.16
[    1s] [48/250] keeping libunistring2-0.9.10-2.11
[    1s] [49/250] keeping libuuid1-2.36.2-2.1
[    1s] [50/250] keeping libverto1-0.3.1-1.16
[    1s] [51/250] keeping libz1-1.2.11-18.16
[    1s] [52/250] keeping libzstd1-1.5.0-1.12
[    1s] [53/250] keeping patch-2.7.6-3.42
[    1s] [54/250] keeping rzsz-0.12.21~rc-4.4
[    1s] [55/250] keeping update-alternatives-1.20.9-1.1
[    1s] [56/250] keeping attr-2.5.1-1.12
[    1s] [57/250] keeping libbrotlidec1-1.0.9-1.4
[    1s] [58/250] keeping libctf-nobfd0-2.36-4.12
[    1s] [59/250] keeping libelf1-0.185-1.1
[    1s] [60/250] keeping libgcrypt20-1.9.3-2.1
[    1s] [61/250] keeping libgdbm_compat4-1.19-1.15
[    1s] [62/250] keeping libglib-2_0-0-2.68.3-4.1
[    1s] [63/250] keeping libidn2-0-2.3.2-1.1
[    1s] [64/250] keeping libisl23-0.24-1.12
[    1s] [65/250] keeping libmpfr6-4.1.0-2.13
[    1s] [66/250] keeping libselinux1-3.2-3.1
[    1s] [67/250] keeping libstdc++6-11.1.1+git536-1.1
[    1s] [68/250] keeping perl-base-5.32.1-1.12
[    1s] [69/250] keeping pkgconf-1.7.4-1.1
[    1s] [70/250] keeping chkstat-1550_20210518-28.14
[    1s] [71/250] keeping libfdisk1-2.36.2-2.1
[    1s] [72/250] keeping libldap-2_4-2-2.4.59-67.1
[    1s] [73/250] keeping libxml2-2-2.9.12-1.1
[    1s] [74/250] keeping libmagic1-5.40-1.13
[    1s] [75/250] keeping build-mkbaselibs-20210120-1.6
[    1s] [76/250] keeping rpm-build-perl-4.16.1.3-3.1
[    1s] [77/250] keeping dwz-0.14-1.12
[    1s] [78/250] keeping file-5.40-1.13
[    1s] [79/250] keeping findutils-4.8.0-2.14
[    1s] [80/250] keeping libgmodule-2_0-0-2.68.3-4.1
[    1s] [81/250] keeping valgrind-3.17.0-2.1
[    1s] [82/250] keeping libasan6-11.1.1+git536-1.1
[    1s] [83/250] keeping libdb-4_8-4.8.30-38.37
[    1s] [84/250] keeping liblsan0-11.1.1+git536-1.1
[    1s] [85/250] keeping libmount1-2.36.2-2.1
[    1s] [86/250] keeping libmpc3-1.2.1-1.14
[    1s] [87/250] keeping libpsl5-0.21.1-1.5
[    1s] [88/250] keeping libtsan0-11.1.1+git536-1.1
[    1s] [89/250] keeping libubsan1-11.1.1+git536-1.1
[    1s] [90/250] keeping tar-1.34-2.1
[    1s] [91/250] keeping libdw1-0.185-1.1
[    1s] [92/250] keeping krb5-1.19.1-4.1
[    1s] [93/250] keeping libsemanage2-3.2-1.14
[    1s] [94/250] keeping libssh4-0.9.5-1.4
[    1s] [95/250] keeping babeltrace-1.5.8-2.6
[    1s] [96/250] keeping cpp11-11.1.1+git536-1.1
[    1s] [97/250] keeping perl-5.32.1-1.12
[    1s] [98/250] keeping brp-check-suse-84.87+git20210706.41c04c1-1.1
[    1s] [99/250] keeping libcurl4-7.78.0-1.1
[    1s] [100/250] keeping libdebuginfod1-0.185-1.1
[    1s] [101/250] keeping terminfo-base-6.2.20210724-24.1
[    1s] [102/250] keeping libncurses6-6.2.20210724-24.1
[    1s] [103/250] keeping libreadline8-8.1-2.13
[    1s] [104/250] keeping ncurses-utils-6.2.20210724-24.1
[    1s] [105/250] keeping nano-5.8-1.1
[    1s] [106/250] keeping bash-5.1.8-1.1
[    1s] [107/250] keeping login_defs-4.8.1-7.1
[    1s] [108/250] keeping sysuser-shadow-3.1-2.1
[    1s] [109/250] keeping cpio-2.13-2.18
[    1s] [110/250] keeping cpp-11-2.1
[    1s] [111/250] keeping diffutils-3.7-5.1
[    1s] [112/250] keeping gzip-1.10-9.1
[    1s] [113/250] keeping hostname-3.23-2.6
[    1s] [114/250] keeping make-4.3-2.34
[    1s] [115/250] keeping which-2.21-4.44
[    1s] [116/250] keeping bzip2-1.0.8-3.1
[    1s] [117/250] keeping gawk-5.1.0-2.18
[    1s] [118/250] keeping grep-3.6-3.1
[    1s] [119/250] keeping pkgconf-pkg-config-1.7.4-1.1
[    1s] [120/250] keeping strace-5.13-1.1
[    1s] [121/250] keeping xz-5.2.5-2.1
[    1s] [122/250] keeping less-590-2.1
[    1s] [123/250] keeping lua54-5.4.3-4.1
[    1s] [124/250] keeping sed-4.8-3.7
[    1s] [125/250] keeping gettext-runtime-0.21-2.16
[    1s] [126/250] keeping iproute2-5.13-1.1
[    1s] [127/250] keeping coreutils-8.32-8.4
[    1s] [128/250] keeping binutils-2.36-4.12
[    1s] [129/250] keeping vim-8.2.3318-1.1
[    1s] [130/250] keeping python38-base-3.8.10-4.1
[    1s] [131/250] keeping compat-usrmerge-build-84.87-3.1
[    1s] [132/250] keeping python38-apipkg-1.5-4.4
[    1s] [133/250] keeping python38-appdirs-1.4.4-29.1
[    1s] [134/250] keeping python38-attrs-21.2.0-33.3
[    1s] [135/250] keeping python38-iniconfig-1.1.1-5.6
[    1s] [136/250] keeping python38-mccabe-0.6.1-26.35
[    1s] [137/250] keeping python38-more-itertools-8.8.0-33.6
[    1s] [138/250] keeping python38-ordered-set-3.1.1-20.1
[    1s] [139/250] keeping python38-pyparsing-2.4.7-81.3
[    1s] [140/250] keeping python38-toml-0.10.2-20.13
[    1s] [141/250] keeping python38-wcwidth-0.2.5-39.2
[    1s] [142/250] keeping systemd-rpm-macros-13-1.1
[    1s] [143/250] keeping libpython3_8-1_0-3.8.10-4.1
[    1s] [144/250] keeping libxcrypt-devel-4.4.23-1.1
[    1s] [145/250] keeping linux-glibc-devel-5.13-1.1
[    1s] [146/250] keeping python38-lazy-object-proxy-1.6.0-20.1
[    1s] [147/250] keeping python38-six-1.16.0-105.1
[    1s] [148/250] keeping python38-wrapt-1.12.1-29.4
[    1s] [149/250] keeping rpm-build-python-4.16.1.3-3.1
[    1s] [150/250] keeping system-group-hardware-20170617-22.1
[    1s] [151/250] keeping glibc-locale-base-2.33-9.1
[    1s] [152/250] keeping libctf0-2.36-4.12
[    1s] [153/250] keeping python38-py-cpuinfo-8.0.0-18.4
[    1s] [154/250] keeping permissions-config-1550_20210518-28.14
[    1s] [155/250] keeping gettext-tools-0.21-2.16
[    1s] [156/250] keeping aaa_base-84.87+git20210727.b447649-1.1
[    1s] [157/250] keeping python36-base-3.6.13-5.1
[    1s] [158/250] keeping python39-base-3.9.6-1.2
[    1s] [159/250] keeping rpm-4.16.1.3-3.1
[    1s] [160/250] keeping aaa_base-malloccheck-84.87+git20210727.b447649-1.1
[    1s] [161/250] keeping glibc-locale-2.33-9.1
[    1s] [162/250] keeping python36-apipkg-1.5-4.4
[    1s] [163/250] keeping python36-appdirs-1.4.4-29.1
[    1s] [164/250] keeping python36-attrs-21.2.0-33.3
[    1s] [165/250] keeping python36-iniconfig-1.1.1-5.6
[    1s] [166/250] keeping python36-mccabe-0.6.1-26.35
[    1s] [167/250] keeping python36-more-itertools-8.8.0-33.6
[    1s] [168/250] keeping python36-ordered-set-3.1.1-20.1
[    1s] [169/250] keeping python36-pyparsing-2.4.7-81.3
[    1s] [170/250] keeping python36-toml-0.10.2-20.13
[    1s] [171/250] keeping python36-typing_extensions-3.10.0.0-30.1
[    1s] [172/250] keeping python36-wcwidth-0.2.5-39.2
[    1s] [173/250] keeping python39-apipkg-1.5-4.4
[    1s] [174/250] keeping python39-appdirs-1.4.4-29.1
[    1s] [175/250] keeping python39-attrs-21.2.0-33.3
[    1s] [176/250] keeping python39-iniconfig-1.1.1-5.6
[    1s] [177/250] keeping python39-mccabe-0.6.1-26.35
[    1s] [178/250] keeping python39-more-itertools-8.8.0-33.6
[    1s] [179/250] keeping python39-ordered-set-3.1.1-20.1
[    1s] [180/250] keeping python39-pyparsing-2.4.7-81.3
[    1s] [181/250] keeping python39-toml-0.10.2-20.13
[    1s] [182/250] keeping python39-wcwidth-0.2.5-39.2
[    1s] [183/250] keeping libpython3_6m1_0-3.6.13-5.1
[    1s] [184/250] keeping libpython3_9-1_0-3.9.6-1.2
[    1s] [185/250] keeping permissions-20210518.1550-28.14
[    1s] [186/250] keeping python36-six-1.16.0-105.1
[    1s] [187/250] keeping python38-execnet-1.8.0-1.3
[    1s] [188/250] keeping python38-zipp-3.5.0-25.2
[    1s] [189/250] keeping python39-lazy-object-proxy-1.6.0-20.1
[    1s] [190/250] keeping python39-six-1.16.0-105.1
[    1s] [191/250] keeping python39-wrapt-1.12.1-29.4
[    1s] [192/250] keeping rpm-config-SUSE-0.g83-1.1
[    1s] [193/250] keeping python36-py-cpuinfo-8.0.0-18.4
[    1s] [194/250] keeping python38-astroid-2.7.1-65.1
[    1s] [195/250] keeping python38-packaging-20.9-51.1
[    1s] [196/250] keeping python38-py-1.10.0-1.4
[    1s] [197/250] keeping python39-py-cpuinfo-8.0.0-18.4
[    1s] [198/250] keeping glibc-devel-2.33-9.1
[    1s] [199/250] keeping librpmbuild9-4.16.1.3-3.1
[    1s] [200/250] keeping build-compare-20200727T175347.d95eb35-1.9
[    1s] [201/250] keeping gdb-10.1-11.1
[    1s] [202/250] keeping python36-execnet-1.8.0-1.3
[    1s] [203/250] keeping python36-zipp-3.5.0-25.2
[    1s] [204/250] keeping python38-importlib-metadata-3.7.2-16.2
[    1s] [205/250] keeping python39-execnet-1.8.0-1.3
[    1s] [206/250] keeping python39-zipp-3.5.0-25.2
[    1s] [207/250] keeping python36-lazy-object-proxy-1.6.0-20.1
[    1s] [208/250] keeping python36-packaging-20.9-51.1
[    1s] [209/250] keeping python36-py-1.10.0-1.4
[    1s] [210/250] keeping python36-typed-ast-1.4.3-34.3
[    1s] [211/250] keeping python36-wrapt-1.12.1-29.4
[    1s] [212/250] keeping python39-astroid-2.7.1-65.1
[    1s] [213/250] keeping python39-packaging-20.9-51.1
[    1s] [214/250] keeping python39-py-1.10.0-1.4
[    1s] [215/250] keeping libutempter0-1.2.0-3.14
[    1s] [216/250] keeping python38-setuptools-57.4.0-201.4
[    1s] [217/250] keeping post-build-checks-84.87+git20210304.df696a0-1.1
[    1s] [218/250] keeping pam-1.5.1-7.1
[    1s] [219/250] keeping rpmlint-mini-1.10-23.26
[    1s] [220/250] keeping gcc11-11.1.1+git536-1.1
[    1s] [221/250] keeping rpmlint-Factory-1.0-99.2
[    1s] [222/250] keeping gcc-11-2.1
[    1s] [223/250] keeping python38-pluggy-0.13.1-1.7
[    1s] [224/250] keeping python39-importlib-metadata-3.7.2-16.2
[    1s] [225/250] keeping python36-importlib-metadata-3.7.2-16.2
[    1s] [226/250] keeping pam_unix-1.5.1-7.1
[    1s] [227/250] keeping python36-astroid-2.7.1-65.1
[    1s] [228/250] keeping python38-isort-5.9.3-87.9
[    1s] [229/250] keeping python36-setuptools-57.4.0-201.4
[    1s] [230/250] keeping python39-setuptools-57.4.0-201.4
[    1s] [231/250] keeping shadow-4.8.1-7.1
[    1s] [232/250] keeping util-linux-2.36.2-2.1
[    1s] [233/250] keeping gcc-PIE-11-2.1
[    1s] [234/250] keeping python36-pluggy-0.13.1-1.7
[    1s] [235/250] keeping python39-pluggy-0.13.1-1.7
[    1s] [236/250] keeping python36-isort-5.9.3-87.9
[    1s] [237/250] keeping python39-isort-5.9.3-87.9
[    1s] [238/250] keeping python38-pytest-6.2.4-1.2
[    1s] [239/250] keeping rpm-build-4.16.1.3-3.1
[    1s] [240/250] keeping python38-pytest-forked-1.3.0-1.4
[    1s] [241/250] keeping python38-pytest-benchmark-3.2.3-3.6
[    1s] [242/250] keeping python36-pytest-6.2.4-1.2
[    1s] [243/250] keeping python39-pytest-6.2.4-1.2
[    1s] [244/250] keeping python36-pytest-forked-1.3.0-1.4
[    1s] [245/250] keeping python39-pytest-forked-1.3.0-1.4
[    1s] [246/250] keeping python38-pytest-xdist-2.2.0-1.5
[    1s] [247/250] keeping python36-pytest-benchmark-3.2.3-3.6
[    1s] [248/250] keeping python39-pytest-benchmark-3.2.3-3.6
[    1s] [249/250] keeping python36-pytest-xdist-2.2.0-1.5
[    1s] [250/250] keeping python39-pytest-xdist-2.2.0-1.5
```

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue. I don't see your version of numpy in the dependency, is that normal ? The error seems to be about numpy.

**Comment by bnavigator** [comment]
Numpy is not installed in the build environment. I don't see it in your test requirements either.

**Comment by bnavigator** [comment]
There must have been some cache involved. When I build in a pristine VM (`osc build --clean`) everything builds fine.

I just noticed the same behavior in astroid.
